mongodb/src/test/java/org/hibernate/ogm/datastore/mongodb/test/gridfs/GridFSTest.java

```diff
@@ -165,10 +188,15 @@ private MongoDatabase getCurrentDB(EntityManager em) {
 		return mongoDBDatastoreProvider.getDatabase();
 	}
 
-	private void assertBlobAreEqual(Blob actual, byte[] expected) {
+	private void assertBlobAreEqual(Blob actual, byte[] expected) throws IOException {
 		try {
 			assertThat( actual.length() ).isEqualTo( expected.length );
-			assertThat( actual.getBytes( 1, expected.length ) ).isEqualTo( expected );
+			InputStream binaryStream = actual.getBinaryStream();
+			ByteArrayOutputStream fullContent = new ByteArrayOutputStream( expected.length );
+			//read full content from blob
+			IOUtils.copy( binaryStream, fullContent );
+
+			assertThat( fullContent.toByteArray() ).isEqualTo( expected );
```

Comment:
What's the benefit of this change compare to the previous one?

Reply:
Instead of adding a new library for a simple conversion. it should be possible to replace this with:
```
	private void assertBlobAreEqual(Blob actual, byte[] expected) {
		try ( InputStream binaryStream = actual.getBinaryStream() ) {
			byte[] actualAsByte = new byte[expected.length];
			DataInputStream stream = new DataInputStream( binaryStream );
			stream.readFully( actualAsByte );
			stream.close();
			assertThat( actualAsByte ).isEqualTo( expected );
		}
		catch (Exception e) {
			throw new RuntimeException( e );
		}
	}
```

Reply:
used. Thanks